packages/core/core.ts

```diff
@@ -271,8 +271,9 @@ function stopFuzzing(
 				`ERROR: Received no error, but expected one of [${expectedErrors}].`,
 			);
 			stopFuzzing(ERROR_UNEXPECTED_CODE);
-		} else if (forceShutdownWithCode === 0) {
-			stopFuzzing(forceShutdownWithCode);
+		} else {
```

Comment:
Calling `stopFuzzing` will kill the node process in the C++ part. In case no error occurred the node process should close down normally, shouldn't it?

Reply:
As far as I know, stopFuzzing(0) gracefully shuts down the fuzzer with exit code 0, so that libfuzzer prints everything it is supposed to on a graceful shutdown.

Reply:
Hmm, strange. Perhaps some caching issues? If this issue comes up again we will take another look!